shared/js/background/classes/ad-click-attribution-policy.js

```diff
@@ -65,15 +77,19 @@ export class AdClickAttributionPolicy {
         const linkFormat = this.getMatchingLinkFormat(resourceURL)
         if (!linkFormat) return
 
-        const adClick = new AdClick(this.navigationExpiration, this.totalExpiration)
+        const adClick = new AdClick(this.navigationExpiration, this.totalExpiration, this.allowlist)
```

Comment:
Nit: We could just pass a reference to `this` in here and *not* store any of these three variables; we can regenerate the `AdClickAttributionPolicy` whenever or have a global to store one in like we do for tab.

Reply:
Do you have a preference? Seems like extra work to me.

Reply:
I think the latter; let's leave it for a follow up actually though :)